utility/utils.py

```diff
@@ -580,7 +581,7 @@ def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, s
         msg['From'] = sender
         msg['To'] = ", ".join(recipients)
         html = create_html_file(run_name=run_name, results_list=results_list, run_time=suite_run_time,
-                                user=trigger_user, run_dir=run_dir)
+                                user=trigger_user, run_dir=run_dir, info=info)
```

Comment:
As the test result data is being used in multiple places, it might require a separate PR align the movement.